crypto/compactcert/structs.go

```diff
@@ -73,3 +81,21 @@ type Cert struct {
 // SortUint64 implements sorting by uint64 keys for
 // canonical encoding of maps in msgpack format.
 type SortUint64 = basics.SortUint64
+
+// StateProofMessage represents the message to be certified.
+type StateProofMessage struct {
```

Comment:
maybe we can move this and the generatefunction to a different file.

Reply:
Moving the generate function might be tricky. 
I've decided to put the Message in a new package in data/stateproof